### PR TITLE
[Documentation][Book] Product Reviews

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | no
| New docs?    | yes
| BC breaks?      | no
| Related tickets | partially #5275 
| License         | MIT

### CODE REVIEW

File: docs/book/product_reviews.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
+.. index::
+    single: Product Reviews
+
+Product Reviews
+===============
+
+Product Reviews are a marketing tool that let your customers to give opinions about the products they buy in your shop.
+They have a ``rating`` and ``comment``.
```
Comment (GSadee) [inline docs/book/product_reviews.rst]: and.. `title`?

Comment (TheMadeleine) [inline docs/book/product_reviews.rst]: Not sure if we need to mention it here :)

File: docs/book/product_reviews.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+    $review->setTitle('My Review');
+    $review->setRating(5);
+    $review->setComment('This product is really great');
+
+Then set it's status to ``new``:
+
+.. code-block:: php
+
+    $review->setStatus(ReviewInterface::STATUS_NEW);
```
Comment (pjedrzejewski) [inline docs/book/product_reviews.rst]: That should be the initial status, we should not be required to set it here.

File: docs/book/product_reviews.rst [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+Then get a customer from the repository, which you would like to make an author of this review.
+
+.. code-block:: php
+
+    $customer = $this->container->get('sylius.repository.customer')->findOneBy(['email' => 'john.doe@test.com']);
+
+    $review->setAuthor($customer);
+
+Remember to add the review to a product and then add it to the repository.
```
Comment (pjedrzejewski) [inline docs/book/product_reviews.rst]: I'd do it a bit differently: set the product on the review and then add it to repository.

**Comment by pjedrzejewski** [comment]
Thanks Magda, nice work! :)

**Pull request merged**